zetta_utils/mazepa/execution.py

```diff
@@ -217,12 +262,9 @@ def submit_ready_tasks(
             task_outcomes={e.payload.task_id: e.payload.outcome for e in task_outcomes}
         )
 
-        # it's not important when to acknowledge outcomes, since
-        # only the single manager node will be pulling from the
-        # outcome queue
-        with ProcessPool(max_workers=num_procs) as pool:
-            for e in task_outcomes:
-                pool.schedule(e.acknowledge_fn)
+    # TODO: Restore multiprocessed acknowledgement when remote
+    for e in task_outcomes:
+        e.acknowledge_fn()
```

Comment:
@nkemnitz might complain about single processed acknowledgement, he has been bottlenecked by this in the past
![image](https://github.com/ZettaAI/zetta_utils/assets/5151706/4c6f3c2c-68ca-4e3c-bed5-7c438dfca096)